apps/etterlatte-saksbehandling-ui/client/src/shared/api/revurdering.ts

```diff
@@ -3,7 +3,11 @@ import { RevurderingInfo } from '~shared/types/RevurderingInfo'
 
 export const lagreRevurderingInfo = (args: {
   behandlingId: string
+  begrunnelse: string
   revurderingInfo: RevurderingInfo
 }): Promise<ApiResponse<void>> => {
-  return apiClient.post(`/revurdering/${args.behandlingId}/revurderinginfo`, { info: args.revurderingInfo })
+  return apiClient.post(`/revurdering/${args.behandlingId}/revurderinginfo`, {
+    begrunnelse: !!args.begrunnelse ? args.begrunnelse : null,
```

Comment:
Hvis den aldri kan være null eller tom så burde det kanskje gjenspeiles i typen her?

Reply:
> Hvis den aldri kan være null eller tom så burde det kanskje gjenspeiles i typen her?

Sant, men denne er nok null/tom for andre revurderingstypar enn adopsjon/omgjøring av farskap/søskenjustering, iallfall per no.